app/index.js

```diff
@@ -6,9 +6,31 @@ const yeoman = require('yeoman-generator');
 const _s = require('underscore.string');
 
 module.exports = yeoman.Base.extend({
+	constructor: function () { // eslint-disable-line babel/object-shorthand
```

Comment:
Am probably missing something here, but why not `constructor()`? It appears to transpile to what you'd expect.

```js
const obj = {
  constructor() {
    console.log(this);
  }
};
```

Output
```js
"use strict";

var obj = {
  constructor: function constructor() {
    console.log(this);
  }
};
```